FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Jyothish Johnson on 08/09/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+final internal class ManagedCache: NSManagedObject {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`, please remove it.
```suggestion
final class ManagedCache: NSManagedObject {
```

Reply:
Thankyou, i have made the changes as mentioned.